src/dask_awkward/core.py

```diff
@@ -808,8 +822,11 @@ def _first_partition(array: Array) -> ak.Array:
         awkward array.
 
     """
-    computed = array.__dask_scheduler__(
-        array.__dask_graph__(), array.__dask_keys__()[0]
+    (computed,) = dask_compute(
```

Comment:
@martindurant thought you might be interested in this.

I ran into a problem with the original implementation we thought up:

```python
parts = array.__dask_graph__()[list(array.__dask_keys__())[0]]
f = parts[0]
args = parts[1:]
return f(*args)
```
because `args` can end up containing uncomputed keys (especially in the case of `from_json` where `read_bytes` is used.

I realized with the addition of culling we can just use `array.partitions[0]` and get the "speedup" we observed from improving this function... but that created infinite recursion because `new_array_object` will always try to compute a meta if it's passed None (and calling `array.partitions[i]` always eventually calls `new_array_object`.. hence the infinite recursion. Anyway... adding the ability to ignore meta (a bit of an "internal" kwarg) fixes everything.